app/src/marketplace/utils/auth.js

```diff
@@ -23,12 +36,13 @@ export const userIsNotAuthenticated = connectedRouterRedirect({
     authenticatingSelector,
     // This sends the user either to the query param route if we have one, or to
     // the landing page if none is specified and the user is already logged in
-    redirectPath: (state, ownProps) => locationHelper.getRedirectQueryParam(ownProps) || '/',
+    redirectPath: (state, ownProps) => locationHelper.getRedirectQueryParam(ownProps) || routes.streams(),
```

Comment:
Defaulting to `/u/streams` now, altho we do have a redirection from `/u` to `/u/streams` now.